sunpy/map/mapbase.py

```diff
@@ -1726,7 +1774,13 @@ def plot(self, annotate=True, axes=None, title=True,
                               SunpyUserWarning)
 
         # Normal plot
-        imshow_args = copy.deepcopy(self.plot_settings)
+        imshow_args = {
+                        'cmap': self.cmap,
+                        'norm': self.norm,
+                        'interpolation': 'nearest',
+                        'origin': 'lower',
+                      }
+
```

Comment:
I think we should add a `imshow_args.update(self.plot_settings)` here. Otherwise, the `interpolation` and `origin` keywords in `plot_settings` (if they are present) will have no effect. As with the `plot_settings` logic in `__init__`, we can just remove this line in 2.1